amqp/definitions.json

```diff
@@ -0,0 +1,50 @@
+{
+    "users": [
+        {
+            "name": "cockpit-tests",
+            "password_hash":"pB0/JnjGQ9tS8/w9gU7Fx8yRsquH54tnX3bufGZ0KQqho5Wz",
+            "hashing_algorithm":"rabbit_password_hashing_sha256",
+            "tags": ""
+        },
+        {
+            "name": "cockpit-amqp",
+            "password_hash":"pB0/JnjGQ9tS8/w9gU7Fx8yRsquH54tnX3bufGZ0KQqho5Wz",
+            "hashing_algorithm":"rabbit_password_hashing_sha256",
+            "tags": ""
+        },
+        {
+            "name": "worky",
+            "password_hash":"pB0/JnjGQ9tS8/w9gU7Fx8yRsquH54tnX3bufGZ0KQqho5Wz",
+            "hashing_algorithm":"rabbit_password_hashing_sha256",
+            "tags": ""
+        }
+    ],
```

Comment:
As discussed on IRC, let's please just keep one user. Unfortunately JSON can't have comments, but please at least mention in the commit  message that the password is never being used (due to SSL auth), and thus a public static password is ok.